### PR TITLE
fix!: Rename webrtc to webrtc-direct and webrtc-w3c to webrtc

### PR DESCRIPTION
**Pull request by mxinden**
We have decided to rename `webrtc-w3c` to `webrtc-private-to-private`. See rational in https://github.com/multiformats/multiaddr/pull/150/#issuecomment-1460912728.

No software has been released using `webrtc-w3c`, thus renaming is not a breaking change. Ignoring the automated `multiaddr` `v11.5.0` release.

See also https://github.com/multiformats/js-multiaddr/pull/309 that introduced `webrtc-w3c`.

### CODE REVIEW

**Comment by mxinden** [comment]
> No software has been released using `webrtc-w3c`, thus renaming is not a breaking change. Ignoring the automated `multiaddr` `v11.5.0` release.

@achingbrain is the assumption correct that this does not pose any issues even though there has been an automated release,namely `v11.5.0`, with `webrtc-w3c`?

**Comment by mxinden** [comment]
Friendly ping @achingbrain, @MarcoPolo and @marten-seemann.

**Comment by MarcoPolo** [comment]
Are we actually using this anywhere? Why is it a breaking change?

**Comment by achingbrain** [comment]
It's breaking because methods that previously accepted `"webrtc-w3c"` as input or returned it will now accept/return `"webrtc-private-to-private"` instead.

**Comment by MarcoPolo** [comment]
Yes, it's technically a breaking change. I see that. Is there anything that is using this that will be affected?

File: src/protocols-table.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const table: Array<[number, number, string, boolean?, boolean?]> = [
   [276, 0, 'p2p-webrtc-direct'],
   [277, 0, 'p2p-stardust'],
   [280, 0, 'webrtc'],
```
Comment (achingbrain) [inline src/protocols-table.ts:23]:
```suggestion
  [280, 0, 'webrtc-direct'],
```

File: src/protocols-table.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const table: Array<[number, number, string, boolean?, boolean?]> = [
...
-  [281, 0, 'webrtc-w3c'],
+  [281, 0, 'webrtc-private-to-private'],
```
Comment (achingbrain) [inline src/protocols-table.ts:24]:
```suggestion
  [281, 0, 'webrtc'],
```

**Review by achingbrain — CHANGES_REQUESTED** [review]
In https://github.com/multiformats/multiaddr/pull/152/files we decided that the browser-to-server version would be called `webrtc-direct` and the browser-to-browser version would just be `webrtc`

**Review by achingbrain — APPROVED** [review]

**Pull request merged**